package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "altis-analytics",
-  "version": "0.0.1",
+  "version": "2.0.0",
```

Comment:
This should probably be `private: true` anyway?

Reply:
TIL